### ISSUE TITLE
Inconsistent status updates

### ISSUE THREAD

**Issue by bartdesign** [SphtKr/homebridge-zway#61]
Hi,

Im using homebridge-zway in combination with a razberry. It works pretty awesome 👍 

There is one little issue i cannot figure out; When i dim a light in the Home app after changing the slider it jumps back to the old value. Then it usually jumps back again to the set value. Sometimes this happens 2 times in a row. I tried changing the polling rate but it doenst seem to make a difference. It looks like it polls before the command is executed. Sometimes this also happens with on/off actions. Any way to make it more stable? Let me know if you need any logs.

**Comment by SphtKr** [comment]
Yes, I've noticed this. I think it's a race condition where the 2-second poll information is coming back as the new value gets sent out...or at least that's one of the ways. I want to get rid of polling eventually with #25 and I hope this will improve this issue.

**Comment by timurvalishev** [comment]
Guys, is there any workaround for this? I'm having this issue, too.

**Comment by pasztorl** [comment]
+1

**Comment by dissar** [comment]
Did anyone find a workaround or something to fix this issue?
I found something interesting;
I have a configuration with two different types of dimmers.
The Everspring AD146: http://www.everspring.com/portfolio-item/ad146-2-wire3-wire-in-wall-dimmer-module/
Fibaro Dimmer 2: https://manuals.fibaro.com/dimmer-2/
In regards to this issue, the everspring dimmer works flawlessly, while the Fibaro dimmer keeps jumping between the new and the previous setting while adjusting the slider.
I wondered  if the issue might be with the zway-server itself, but all dimmers work fine when using the Z-Wave Smart Home UI. I have not tried adjusting with any other plugins.
I did however test a Z-Way App called 'Dimmable Scene', and that fixed it. But it's not ideal since it doesn't have a status feedback (goes back to 0% after adjustment). It also seems to dim a little bit down first before dimming up.
The app 'Group devices' did not help.

**Comment by hubert3** [comment]
I'm having this issue with an Aeotec Z-Wave Plus RGB Bulb 5

RGB colour control works perfectly through the Z-Way web interface, but through the Home app UI on phone or Mac desktop via homebridge-zway, the colour I pick from the wheel is initially set and then it jumps back to another colour

Wish I could entirely disable polling for the current value, I care more about the control simply working correctly for setting a new colour than reflecting the currently set device value in the UI
